src/ping.c

```diff
@@ -468,6 +475,25 @@ static int ping_config(const char *key, const char *value) /* {{{ */
     hl->latency_squared = 0.0;
     hl->next = hostlist_head;
     hostlist_head = hl;
+  } else if (strcasecmp(key, "AddressFamily") == 0) {
+    char *af = NULL;
+    int status = config_set_string(key, &af, value);
+    if (status != 0)
+      return status;
+
+    if (strncmp(af, "any", 3) == 0) {
+      ping_af = AF_UNSPEC;
+    } else if (strncmp(af, "ipv4", 4) == 0) {
+      ping_af = AF_INET;
+    } else if (strncmp(af, "ipv6", 4) == 0) {
+      ping_af = AF_INET6;
+    } else {
+      ERROR("ping plugin: Bad address family: %s", af);
```

Comment:
Rest of this function uses `WARNING` on 'invalid'/unsupported option values.
I think code should be consistent. What do you think about this?

Reply:
I overlooked that. Consistency is important, and I see that collectd is trying to continue as it can. Updating....